pandas/core/frame.py

```diff
@@ -5150,7 +5150,7 @@ def drop(
         ...                         [1, 0.8], [0.3, 0.2]])
         >>> df
                         big     small
-        lama    speed   45.0    30.0
+        llama    speed   45.0    30.0
```

Comment:
these aren't aligned anymore

I'd suggest either:
- anytime you change lama to llama, fixup the alignment
- or, just keep the `codespell` update out of this PR

Reply:
and this